### ISSUE TITLE
Display Version and last Change on Doc Pages  

### ISSUE THREAD

**Issue by Ion3** [openhab/openhab-docs#1469]
I have sometimes issues using information that still refers to Version 2.x. Wouldn't it make sense to show last update of the page and the referenced software version?

**Comment by Confectrician** [comment]
When `latest` is choosen, it should include only contents that are valid in the latest version. (So openHAB 3 currently.)
When you find something that is related to a 2.x version and **not valid** for openHAB 3, we should remove this part.

Showing a "last updated" state per page will be a huge effort probably an i am not sure if it will bring the expected benefits compared to that effort.

**Comment by Ion3** [comment]
Example:

[This](https://www.openhab.org/docs/configuration/persistence.html):
![latest_Persistance](https://user-images.githubusercontent.com/6546080/106053311-f68d4800-60ea-11eb-8bba-2d86e5375c3c.png)

is not matching your [main](https://github.com/openhab/openhab-docs/blob/main/configuration/persistence.md). It has still content of 2.x, as 3.0 does not use the paper UI.

**Comment by Confectrician** [comment]
Thanks for the hint that seems to be a different problem.
https://ci.openhab.org/view/Documentation%20(3.x)/

Seems that we have some problems with our website builds in general.
The wrong paper ui reference has been fixed.

But in general we should then remove such references if they appear.
I have bulk edited many files for terms like paper ui but it is possible that there are still some references out there with another grammar.

**Comment by Ion3** [comment]
Thank you Jerome,

shouldn't we lower the entry barriers for the user? e.g.: [Units Of Measurement](https://github.com/openhab/openhab-docs/blob/main/concepts/units-of-measurement.md) it sounds great but I personally have no clue how to use it in the main UI.

I have a sensor with measures power and energy. To show me power, I go to the item metadata and enter a state description with the pattern "%.0f W". How can the framework now assist me in converting it to kW when the value grows? 

The same more practical example as a lot of newbies have a thermometer. In my case, I did not need to enter a state description, as the semantic definition recognizes it to be a thermometer and places °C behind. By default it shows two decimal places. If I want to change this, I have to enter a state description in the main UI. Entering "%.1f" gives me one decimal place but removes °C. Therefor I have to enter "%.1f °C". Does openHAB still recognizes it to be Celsius? Where in the main UI can the conversion to Fahrenheit be done? I think these are the questions that need to be answered in the documentation to get the community growing, remove repetitive questions in the forum and increase motivation and success for beginners. 

I would love to help but for me the entry barrier is to high. I am struggling on all ends. I am on the verge of giving up.

You experts, please change the perspective: Set yourself into the [DAU](https://de.wikipedia.org/wiki/D%C3%BCmmster_anzunehmender_User) and teach him what he needs to know to get success with openHAB. Try not to reuse old documentation by bulk editing and exchanging terms. Use be benefit arising from the visual user interface.

Thank you for all your hard work and best greetings,
Timo

**Comment by Confectrician** [comment]
Hey Timo,

Those are many topics and they are a bit beyond the topic of this issue.

Main UI:
You are right, we are late with suitable docs about this.
https://github.com/openhab/openhab-docs/pull/1464 is aiming to attack this problem although it is more meant as a reference. (Still nothing on DAU level, to speak in your terms. :) )

Did the Getting started tutorial bring any benefits for you?
We have a whole series of discussions in our forum where the current tutorial state is based on.
Wo we should see where we can do improvements. (And also complete the tutorial for sure. It's still under construction)

> Try not to reuse old documentation by bulk editing and exchanging terms.

To be fair: This wasn't the case at all.
We did bulk edits "where possible", like many articles referenced to settings in the paper ui
which are simply also available in the new UI.

But we also did completely new articles and started the getting started tutorial (a bit late for sure).


> I would love to help but for me the entry barrier is to high. I am struggling on all ends. I am on the verge of giving up.

Help with documentation?
If you tell me where you are struggling i can help or at least point you to additional information for our docs.
